src/classes/Store.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Store;
+use Store\Category as Category;
+use \PDO;
+use Store\interfaces\GetCategories as GetCategories;
+
+class Store implements GetCategories{
```

Comment:
If we are enforcing a GetCategories interface, should we do the same for getProducts?